runBuild.sh

```diff
@@ -1,4 +1,4 @@
-DEV_DIR=/c/Dev_perso/boids
+DEV_DIR="/c/Users/Vol po mon pc esti/Documents/GitHub/Boids"
```

Comment:
We need to put this script in .gitIgnore, it won't be part of git anymore and anyone will be able to set the path as she/he wants.